benchmarks/benchmarks.vcxproj

```diff
@@ -9,13 +9,15 @@
   <Import Project="$(SolutionDir)principia.props" />
   <ItemGroup>
     <ClCompile Include="..\base\status.cpp" />
+    <ClCompile Include="..\ksp_plugin\planetarium.cpp" />
     <ClCompile Include="dynamic_frame.cpp" />
     <ClCompile Include="embedded_explicit_runge_kutta_nyström_integrator.cpp" />
     <ClCompile Include="ephemeris.cpp" />
     <ClCompile Include="hexadecimal.cpp" />
     <ClCompile Include="main.cpp" />
     <ClCompile Include="newhall.cpp" />
     <ClCompile Include="perspective.cpp" />
+    <ClCompile Include="planetarium_benchmarks.cpp" />
```

Comment:
In general we don't call benchmarks `_benchmarks`.

Reply:
Yes, but if there are two translation units with the same name in the same project MSVC complains that the object files have the same name.